runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -3,15 +3,48 @@
  */
 package akka.grpc
 
-import scala.concurrent.duration.Duration
+import akka.actor.ActorSystem
+import com.typesafe.config.Config
 
+import scala.concurrent.duration.Duration
 import io.grpc.CallCredentials
 
 object GrpcClientSettings {
   /** Scala API */
   def apply(host: String, port: Int): GrpcClientSettings =
     new GrpcClientSettings(host, port)
 
+  /** Scala API */
+  def apply(serviceName: String, sys: ActorSystem): GrpcClientSettings = {
+    val akkaGrpcClientConfig = sys.settings.config.getConfig("akka.grpc.client")
```

Comment:
I prefer two steps so that any config section can be used

```
def apply(serviceName: String, sys: ActorSystem): GrpcClientSettings =
  apply(serviceName, sys.settings.config.getConfig("akka.grpc.client"))

def apply(serviceName: String, config: Config): GrpcClientSettings = ...
```